src/main/resources/assets/iris/lang/ru_ru.json

```
@@ -6,9 +6,12 @@
   "iris.shaders.debug.disabled": "Отладка отключена.",
   "iris.shaders.debug.failure": "Невозможно включить отладку, ваш компьютер не поддерживает необходимые функции.",
   "iris.shaders.debug.restart": "Отсутствует отладочный контекст. Перезапустите игру, чтобы включить отладку.",
+  "iris.load.failure.shader": "Не удалось загрузить набор шейдеров! Пожалуйста, сообщите об ошибке разработчику шейдера.",
+  "iris.load.failure.generic": "Iris столкнулся с проблемой при попытке загрузить шейдеры; пожалуйста, сообщите об этом разработчикам Iris.",
   "iris.shaders.ssbofailure": "Запрашиваемая шейдером функция (SSBO) не поддерживается вашим ГП, шейдер может работать неправильно.",
   "iris.keybind.reload": "Перезагрузить шейдеры",
   "iris.keybind.shaderPackSelection": "Список наборов шейдеров",
+  "iris.keybind.wireframe": "Вайрфрейм (только SP)",
```

```suggestion
  "iris.keybind.wireframe": "Полигональная сетка (только локальные миры)",
```

Wireframe – это режим отображения сетки модели местности, транслитом его переводить неуместно и бессмысленно. В русском языке есть соответствующий термин в компьютерной графике, `полигональная сетка`.
Также SP в данном контексте означает "Singleplayer", или одиночная игра. Это ограничение на включение режима просмотра сетки было введено из-за того, что режим просмотра сетки позволяет видеть то, что игрок видеть не должен, и включить его можно только при игре в локальном мире. Возможно также использовать вариант `(только одиночная игра)`.

---

Да, я хотел это уточнить, но часть комментария не отправилась.